.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -9,13 +9,10 @@
 ### Preview link
 <!-- Impacted pages preview links-->
 
-<!-- This is the base preview link. This currently only works if you are in the Datadog organization and working off of a branch - it will not work with a fork. 
+<!-- This is the base preview link. This currently only works if you are in the Datadog organization and working off of a branch - it will not work with a fork.
```

Comment:
```suggestion
<!-- This is the base preview link. This currently only works if you are in the Datadog organization and working off of a branch - it will not work with a fork. -->
```